homeassistant/components/roon/manifest.json

```diff
@@ -6,5 +6,5 @@
   "documentation": "https://www.home-assistant.io/integrations/roon",
   "iot_class": "local_push",
   "loggers": ["roonapi"],
-  "requirements": ["roonapi==0.1.3"]
+  "requirements": ["roonapi==0.1.4"]
```

Comment:
It's always tempting to combine PRs...

Since the library bump contains bugfixes I'd be inclined to make the library bump a preliminary PR so it can be cheery-picked if needed.

Reply:
Got it. 

Library update is now in https://github.com/home-assistant/core/pull/89124

I've left it here too (just to avoid testing my github skills) - but can remove if needed.

Obviously this PR needs the library PR to be merged first.